index.bs

```diff
@@ -1079,8 +1079,8 @@ associated.
     };
 </pre>
 
-Clients may communicate with authenticators using a variety of mechanisms. For example, a client may use a platform-specific
-API to communicate with an authenticator which is physically bound to a platform. On the other hand, a client may use a
+Clients MAY communicate with authenticators using a variety of mechanisms. For example, a client MAY use a platform-specific
```

Comment:
This is descriptive - not normative language. I would suggest changing these "may"s to "can"s.

Reply:
updated to can